dashboard/public/frequency/frequency.html

```diff
@@ -4,6 +4,8 @@
 
 <script type="text/javascript" src="/frequency/frequency.js"></script>
 
+<div id="bubble"></div>
```

Comment:
Well, we're putting the instructions in as the default input.

The question is whether or not that's what we want to be doing (I vote yes), and, if so, whether we want to have something on the page you can click on to bring the markdown instructions back up. And, if so, what that should be.